ServiceManager/GUI/main_window.py

```diff
@@ -0,0 +1,629 @@
+        self.service_log_show_lines_spinbox.setEnabled(enabled)
+    # endregion
+
+    # region Service Status
+    def update_status_title(self, text: str):
+        self.service_status_title.setText(text)
+
+    def update_status_style(self, stylesheet: str):
+        self.service_status_title.setStyleSheet(stylesheet)
+
+    def update_service_details(self, service: dict):
+        self.service_details_name.setText(service['name'])
+        self.service_details_status.setText(service['status'])
+        self.service_details_pid.setText(f"{service['pid']}")
+        self.service_details_startup.setText(service['start_type'])
+        self.service_details_description.setText(service['description'])
+        self.service_details_username.setText(service['username'])
+        self.service_details_binpath.setText(service['binpath'])
+
+    def update_service_control_btns(self, service_status):
+        if service_status == 'running':
+            self.btn_service_start.setEnabled(False)
+            self.btn_service_stop.setEnabled(True)
+            self.btn_service_restart.setEnabled(True)
+        elif service_status == 'stopped':
+            self.btn_service_start.setEnabled(True)
+            self.btn_service_stop.setEnabled(False)
+            self.btn_service_restart.setEnabled(False)
+        elif service_status is None or service_status == 'not found':
+            self.btn_service_start.setEnabled(False)
+            self.btn_service_stop.setEnabled(False)
+            self.btn_service_restart.setEnabled(False)
+        else:
+            self.btn_service_start.setEnabled(True)
+            self.btn_service_stop.setEnabled(True)
+            self.btn_service_restart.setEnabled(True)
+    # endregion
+    # endregion
+
+
+class DeleteConfigsMessageBox(QMessageBox):
+    """ Message Box on deleting configuration entries. Display a list of entries to be deleted and confirm button. """
+    def __init__(self, obj_list: list):
+        super().__init__()
+
+        self.setWindowTitle('Delete configuration')
+        self.setText(f'Deleting the following configuration entries.\n'
+                     f'Are you sure?\n\n\n')
+        self.setIcon(QMessageBox.Warning)
+
+        self.list_ = QListWidget(self)
+        self.list_.move(30, 70)
+        self.list_.resize(450, 120)
+        self.list_.addItems(obj_list)
+
+        self.addButton(QMessageBox.Ok)
+        ok_btn = self.button(QMessageBox.Ok)
+        ok_btn.setText('Delete')
+        self.addButton(QMessageBox.Cancel)
+        self.setDefaultButton(QMessageBox.Cancel)
+
+    def event(self, e):
+        result = QMessageBox.event(self, e)
+        self.setMinimumWidth(0)
+        self.setMaximumWidth(16777215)
+        self.setMinimumHeight(0)
+        self.setMaximumHeight(16777215)
+        self.setSizePolicy(
+            QSizePolicy.Expanding,
+            QSizePolicy.Expanding
+        )
+        self.resize(500, 250)
+
+        return result
```

Comment:
**should**: Could you add a comment about why this resizing is being done on `event`?